genesis-tools/genesis-populate/src/main.rs

```diff
@@ -26,15 +24,5 @@ fn main() {
         .value_of("additional-accounts-num")
         .map(|x| x.parse::<u64>().expect("Failed to parse number of additional accounts."))
         .unwrap();
-    let near_config = load_config(home_dir);
-
-    let store = create_store(&get_store_path(home_dir));
-    GenesisBuilder::from_config_and_store(home_dir, Arc::new(near_config.genesis), store)
-        .add_additional_accounts(additional_accounts_num)
-        .add_additional_accounts_contract(near_test_contracts::tiny_contract().to_vec())
-        .print_progress()
-        .build()
-        .unwrap()
-        .dump_state()
-        .unwrap();
+    prepare_and_dump_state(home_dir, additional_accounts_num);
```

Comment:
Rather than extracting this to a function, I'd just copy-pasted this builder invocation to params-estimator. 

While syntactically the two pieces of code will be duplicate, I don't think they *semantically* do the same thing. Meaning theat, I can imagine the two pieces of code evolving differently -- if params-estimacro populates genesis itself, it's no longer a requirement that genesis-populate tool does exactly the same thing. 

I can imagine us changing the `print_progress` logic, or changing the test contract we use. It would be cool if we could do that in params-estimatro without modifying genesis-populate code.

One more considiration here is that this code doesn't contain any "logic" -- there are no `if`s or other control flow, there's just a straightforward sequence of API calls. There's little benefit in abstracting that out.